Tests/APITesters/CustomEntitlementComputationSwiftAPITester/SwiftAPITester/PresentedOfferingContextAPI.swift

```diff
@@ -14,5 +14,5 @@ func checkPresentedOfferingContextAPI(context: PresentedOfferingContext! = nil)
 }
 
 private func checkCreatePresentedOfferingContextAPI() {
-    let _: PresentedOfferingContext = .init(offeringIdentifier: "")
+    let _: PresentedOfferingContext = .init(offeringIdentifier: "", placementIdentifier: "")
```

Comment:
We should have API tests without the new parameter as well, to make sure we don't break it.